sass/directives/_carousels.scss

```diff
@@ -88,6 +88,63 @@
   }
 }
 
+@mixin resource-header {
+  h2, p, h3, h1 {
+    display: inline-block;
+  }
+
+  nav {
+    display: inline-block;
+    float: right;
+  }
+}
+
+@mixin resource-carousel-arrow {
+  z-index: z($z-context, page_content, $page_content, carousel__arrow);
```

Comment:
Style-base doesn't know what these variables are. Ways to work around it

Tooltips (uses mixin):
Pass in a placeholder z-index: https://github.com/cb-talent-development/employer-style-base/blob/topic/EM-899-Job-Optimizations-2/sass/directives/_tooltips.scss#L1
Overwrite it on Employer: https://github.com/cbdr/employer/blob/topic/EM-899-Jobs-Optimizations-2/app/assets/stylesheets/layouts/_common.scss#L567-L568

CTA (uses classes):
Set a temporary z-index: https://github.com/cb-talent-development/employer-style-base/pull/69/files#diff-074123b636e5df8842bec33396639af8R8
Overwrite it in Employer: https://github.com/cbdr/employer/pull/675/files#diff-7d5253286e5b741a03b8622687a914d2R2